op-docker/coordinator/Dockerfile

```diff
@@ -17,12 +17,40 @@ RUN foundryup
 COPY --from=base-node /usr/local /usr/local
 
 WORKDIR /shared-optimism
-RUN git clone --recurse-submodules https://github.com/ethereum-optimism/optimism.git .
-RUN git checkout v1.2.0
+RUN git clone --branch v1.2.0 --depth 1 --recurse-submodules https://github.com/ethereum-optimism/optimism.git .
 RUN pnpm install 
 RUN pnpm build
 
+# Install ssh client
+RUN apt-get install -y openssh-client
```

Comment:
User must setup an ssh key with github that's passed into the image. This is due to the contracts repo being private